src/diffusers/pipelines/unclip/pipeline_unclip.py

```diff
@@ -0,0 +1,502 @@
+# TODO License
+
+from typing import Callable, List, Optional, Union
+
+import torch
+from torch import nn
+from torch.nn import functional as F
+
+import torchvision.transforms.functional as TVF
```

Comment:
we have no dependency on torchvision - this has to be removed eventually

Reply:
Saw in the dependencies table / setup.py and thought was ok! The function we use is just a wrapper around torch.nn.functional.interpolate so easy to remove 👍

Reply:
done!